src/Workspaces/Remote/Core/Microsoft.CodeAnalysis.Remote.Workspaces.csproj

```diff
@@ -4,7 +4,7 @@
   <PropertyGroup>
     <OutputType>Library</OutputType>
     <RootNamespace>Microsoft.CodeAnalysis.Remote</RootNamespace>
-    <TargetFrameworks>netcoreapp3.1;netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>net6.0;netstandard2.0</TargetFrameworks>
```

Comment:
❓ Do we gain anything concrete from changing this one?

Reply:
Not currently, other then better null annotation checks, but I don't see why we would want to target old .NET Core.

Reply:
I've never been a fan of updating TFMs in libraries for the sake of updating TFMs. We should only update at the point where we have a clear benefit coming from the new version that we can't easily have otherwise, and at that point we should seriously consider multi-targeting.

Reply:
I agree for libraries that are "public" like Microsoft.CodeAnalysis. However, Remote.Workspaces are not the case.